packages/hello-gsm/src/pages/calculator/index.tsx

```diff
@@ -50,6 +51,21 @@ const getInfo = async (accessToken: string) => {
  * 둘다 없다면 로그인 페이지로 이동
  */
 export const getServerSideProps: GetServerSideProps = async ctx => {
+  // 접수 기간이 아니면 페이지 접근 불가
+  if (
+    !(
+      new Date() >= new Date('2022-10-17 9:00') &&
+      new Date() <= new Date('2022-10-20 17:00')
+    )
+  ) {
```

Comment:
```suggestion
const acceptable =
    new Date() >= new Date('2022-10-17 9:00') &&
    new Date() <= new Date('2022-10-20 17:00');

  // 접수 기간이 아니면 페이지 접근 불가
  if (!acceptable) {
```

이런 식으로 조건식을 변수로 사용하면 가독성이 더 좋을 것 같습니다!

Reply:
오오 그럼 저 변수를 shared 디렉토리에 공유하여 여러 페이지에서 사용할 수 있도록 변경해보겠습니다!